architecture-building-systems/CityEnergyAnalyst
# Fixing the pyliburo / radiation-daysim geometry problems

This PR addresses the problems with pyliburo and radiation-daysim geometry as discussed in #1620 (current pyliburo creates different daysim geometry as compared to previous version).

It turns out that porting to py4design is not that hard - took less time than spent so far on getting our fork of pyliburo to work... 

@gabriel-happle could you check this please?

To test:
- create a new conda environment (e.g. `conda env create -n 2018.09.20`)
- activate conda environment (e.g. `activate 2018.09.20`)
- pip install the CityEnergyAnalyst to that environment (`pip install -e .`)
- run radiation (`cea radiation-daysim`)
- compare the buildings, top two data rows of `B01_geometry.csv` should look like this:

```
AREA_m2,BUILDING,SURFACE,TYPE,Xcoor,Xdir,Ycoor,Ydir,Zcoor,Zdir,orientation
9.89061782103,B01,srf0,walls,463110.744064,-0.999801492748,5225354.71161,0.019924233953,423.98029706,-0.0,west
5.50686294303,B01,srf1,walls,463110.744064,-0.999801492748,5225354.71161,0.019924233953,423.584561735,-0.0,west
```

If you'd like to run from PyCharm, make sure to set the project interpreter to the new conda environment.
